package.json

```diff
@@ -29,14 +29,14 @@
     "@storybook/svelte": "^7.1.0-alpha.6",
     "@storybook/sveltekit": "^7.1.0-alpha.6",
     "@storybook/testing-library": "^0.1.0",
-    "@sveltejs/adapter-auto": "^2.0.0",
+    "@sveltejs/adapter-vercel": "^2.4.2",
```

Comment:
Yeah, Vercel outputs a warning saying the build will be faster if it doesn't have to autodetect the adapter. I'm skeptical of that actually being all that slow but I prefer this if only so that we don't see that warning anymore.